packages/core/src/Button/Button.styles.ts

```diff
@@ -46,8 +40,8 @@ export const { staticClasses, useClasses } = createClasses("HvButton", {
   focusVisible: {},
   disabled: {
     color: theme.colors.secondary_60,
-    borderColor: theme.colors.atmo4,
-    backgroundColor: theme.colors.atmo3,
+    border: "none",
+    backgroundColor: "transparent",
```

Comment:
If disabled buttons always have a transparent `backgroundColor`, we should be able to increase the specificity here, instead in the `root` and most variant styles, right?